app/js/preferences/elicitationTradeOffDirective.html

```diff
@@ -33,5 +25,13 @@
           <td class="text-centered">{{secondaryCriterionValue.secondValue | number}}</td>
         </tr>
       </tbody>
+    </table>
+  </div>
+  <div class="cell large-6 medium-6" ng-if="!weight.value && weight.value !== 0">
+    <i class="fa fa-spinner fa-2x fa-spin"></i>
+  </div>
+  <div class="cell large-6 medium-6">
+    <elicitation-trade-off-plot ng-if="weight.value" coord-ranges="coordRanges" criteria="criteria" data="data"
```

Comment:
i think if you want to be complementary to `!weight.value && weight.value !== 0` you should add the zero-check to the `ng-if` here too.